src/utils.ts

```diff
@@ -276,28 +277,26 @@ export const fromBase64 = (value: string) => {
   }
 };
 
-export const getSuccessRateThresholdColor = (value: number | undefined) => {
-  if (value === undefined) {
-    return config.theme2.colors.text.disabled;
-  }
-  if (value >= 99.5) {
-    return config.theme2.colors.success.main;
-  }
-  if (value >= 99) {
-    return config.theme2.colors.warning.main;
-  }
-  return config.theme2.colors.error.main;
+export const getSuccessRateThresholdColor = (
+  thresholds: ThresholdSettings,
+  key: 'reachability' | 'uptime' | 'latency',
+  compareValue: number
+) => {
+  return compareValue < thresholds[key].lowerLimit
```

Comment:
Do you mind busting this out into `if` statements? Nested ternaries make my brain hurt